HCCGo/app/js/jobSubmissionCtrl.js

```diff
@@ -146,7 +146,7 @@ jobSubmissionModule.controller('jobSubmissionCtrl', ['$scope', '$log', '$timeout
 
     // Separate SBATCH options from commands
     job.commands = editor.getValue();
-    var other = editor.getValue().split("\n");
+    var other = editor.getValue().split(/\r?\n/);
```

Comment:
This is only for the jobFile, they are stored in job history without the removal of the "\r", I forgot to change that.